static/src/js/util/emptyAccessMethod.js

```diff
@@ -0,0 +1,31 @@
+const downloadMethod = {
+  download: {
+    type: 'download'
+  }
+}
+
+// TODO
+const echoOrderMethod = {
+  echoOrder: {
+    type: 'ECHO_ORDER'
+  }
+}
+// const opendapMethod
+// const egiMethod
+
+/**
+ * Returns an empty access method based on the provided methodName
+ * @param {string} methodName Access method name
+ */
+const emptyAccessMethod = (methodName) => {
+  switch (methodName) {
+    case 'download':
+      return downloadMethod
+    case 'echoOrder':
```

Comment:
It appears as though the code is looking for downcased JSON friendly strings here and you already have checks for `esi` and `opendap` but they aren't represented here. While we don't need them because the functionality isn't there yet they could be here if it's returning an empty object or something -- I think that'd be better than nothing.